client/src/common/InitialPageForm.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import {
+  TextField,
+  Paper,
+  Typography,
+  makeStyles,
+  Button
+} from "@material-ui/core";
+
+//CSS styles
+const useStyles = makeStyles({
+  container: {
+    width: "490px",
+    margin: "40px auto 0 auto",
+    padding: "2rem 4rem 2rem 4rem",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  formTitle: {
+    marginBottom: "20px"
+  },
+  button: {
+    width: "10rem",
+    margin: "1rem 0 2rem 0"
+  },
+  textField: {
+    marginTop: "10px"
+  }
+});
+
+const InitialPageForm = ({
```

Comment:
Maybe call it PaperForm (since it could be used for any form surrounded by a Paper)